operations/kyverno/Kyverno.md

```diff
@@ -0,0 +1,298 @@
+# Kyverno policy management
+
+[Kyverno](https://kyverno.io/) is a policy engine designed specifically for Kubernetes.
+
+Kyverno allows cluster administrators to manage environment specific configurations independently of workload configurations and enforce configuration best practices for their clusters.
+
+Kyverno can be used to scan existing workloads for best practices, or can be used to enforce best practices by blocking or mutating API requests.
+
+Kyverno enables administrators to do the following:
+* Manage policies as Kubernetes resources.
+* Validate, mutate, and generate resource configurations.
+* Select resources based on labels and wildcards.
+* Block non-conformant resources using admission controls, or report policy violations.
+* View policy enforcement as events.
+* Scan existing resources for violations.
+
+Kyverno policies implement the various levels of Kubernetes [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) for CSM services.
+
+The policies are minimally restrictive and enforces the best practices for pods. The policies make sure the following values are set for workloads (if not present).
+
+```text
+securityContext:
+  allowPrivilegeEscalation: false
+  privileged: false
+  runAsUser: 65534
+  runAsNonRoot: true
+  runAsGroup: 65534
+```
+
+Currently, [mutation](#mutation) and [validation](#validation) policies are enforced for the network services such as load balancer and virtualservice.
+
+## Mutation
+
+Mutation policies are applied in the admission controller while creating pods.
+
+It mutates the manifest of respective workloads before creating it so that when resource comes up, it will abide by the policy constraints.
+
+### Example
+
+1. Create a sample policy.
+	```text
```

Comment:
General note that the code blocks should all line up with the first word of a step.
For example:

```
1. Create a sample policy.

   "```bash"
   example output
   "```"
```

Reply:
It's already aligned

Reply:
The discrepancy is due to the different editor I was using. I have corrected it now!